python-sdk/docs/astro/sql/operators/load_file.rst

```diff
@@ -221,6 +221,17 @@ Loading to MS SQL
 .. note::
    We do not support loading Unicode data to SQL Server due to limitations on the underlying ``pymssql`` library
 
+Loading to Duckdb
+~~~~~~~~~~~~~~~~~
+
+``load_file`` can load data to duckdb. If the database file is not specified in connection, it will be created
+(the file extension may be .db, .duckdb, or anything else). The special value :memory: (the default) can be used to
+create an in-memory database. Note that for an in-memory database no data is persisted to disk (i.e. all data is lost when you exit the Python process).
```

Comment:
what's the behavior like if you load a file into duckdb and the next task gets executed on a different worker? somewhat separately, what's the behavior like for `load_file` with sqlite? I'm a bit concerned about the case where things get executed on separate workers if you're storing data in tmp storage or in memory

Reply:
Users have the option to set a host if they want to use shared database file locations across workers. This is similar to the sqlite database.

With `:memory:` no data is persisted to disk (i.e. all data is lost when you exit the Python process).